bundles/org.openhab.binding.knx/README.md

```diff
@@ -235,18 +235,21 @@ For UoM support see the explanations of the `number` channel.
 #### Group Address Notation
 
 ```text
-<config>="[<dpt>:][<]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
+<config>="[<dpt>:][<>]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
 ```
 
 where parts in brackets `[]` denote optional information.
 
+**Each configuration parameter has a `mainGA` where commands are written to and optionally several `listeningGA`s.**
+
+`mainGA` also listens to incoming packets, unless prefixed with a `>` character.
+This is recommended if you have dedicated status group addresses which are added as `listeningGA`s.
+
 The optional `<` sign tells whether the group address of the datapoint accepts read requests on the KNX bus (it does, if the sign is there).
-All group addresses marked with `<` are read by openHAB during startup.
+Group addresses marked with `<` are read by openHAB during startup, but typically this is useful for only one group address per definition.
 With `*-control` channels, the state is not owned by any device on the KNX bus, therefore no read requests will be sent by the binding, i.e. `<` signs will be ignored for them.
 
-Each configuration parameter has a `mainGA` where commands are written to and optionally several `listeningGA`s.
-
-The `dpt` element is optional. If omitted, the corresponding default value will be used (see the channel descriptions above).
+The element `dpt` is optional. If omitted, the corresponding default value will be used (see the channel descriptions above).
```

Comment:
I wonder if we should change that. A lot of issues especially with DPT9 would not have occurred if the DPT was mandatory. I would prefer to remove the "optional" brackets around the DPT and change this sentence as suggested.

```suggestion
The element `dpt` is highly recommended and change to a mandatory element in future versions.
If omitted, the corresponding default value will be used (see the channel descriptions above). 
```

The code can be like it is now. In the next iteration (i.e. 4.2) we could print a warning for a missing DPT and remove support for "default" DPTs earliest in 4.3.